src/components/notices-control-view/Notice.vue

```diff
@@ -63,7 +63,9 @@ export default {
   },
   components: { Button },
   created() {
-    this.$store.dispatch('notices/getNoticeById', { _id: this.noticeId });
+    if (!this.title) {
```

Comment:
WHy?

Reply:
If we have title -> we have all data for current notice -> we do not need to load this again